controllers/binding/snapshotenvironmentbinding_adapter.go

```diff
@@ -163,6 +165,7 @@ func (a *Adapter) EnsureEphemeralEnvironmentsCleanedUp() (controller.OperationRe
 	if err != nil {
 		return controller.RequeueWithError(fmt.Errorf("failed to update snapshot test status: %w", err))
 	}
+	metrics.RegisterSEBFailedDeployment()
```

Comment:
IMO this should be in ensure function whcih triggers integration test, that's when SEB is successfully deployed

Reply:
nevermind, I'm stupid and I cannot read